entrypoint.sh

```diff
@@ -11,7 +11,16 @@ process_input(){
         my_dir="./test_data"
     fi
 
-    if [ -n "$input_paths" ]; then
+    severity_mode="$(echo $severity_mode | tr '[:upper:]' '[:lower:]')"
+
+    if [[ "$severity_mode" != "style" && "$severity_mode" != "info" && "$severity_mode" != "warning" && "$severity_mode" != "error" ]]; then
```

Comment:
If we set a default value for severity_mode as I discussed above, this if statement can be written as 
if [[ "$severity_mode" != "style" && "$severity_mode" != "info" && "$severity_mode" != "warning" && "$severity_mode" != "error" ]]; then
        echo "Error setting unknown severity mode. Defaulting severity mode to style."
        severity_mode="style"
    fi